docs/en/latest/how-to-build.md

```diff
@@ -204,23 +204,30 @@ Run the `apisix help` command to see the returned results and get commands and d
 
 ```shell
 # more actions find by `help`
-$ apisix help
+apisix help
 ```
 
 ## Step 4: Run Test Cases
 
 1. Install `cpanminus`, the package manager for `perl`.
 
+```shell
+curl -L https://cpanmin.us/ -o cpanm
```

Comment:
I have reconsidered it.
Maybe putting a link to the official installation is better?
https://metacpan.org/pod/App::cpanminus#INSTALLATION

Reply:
That's a good idea, I have changed.